Pyrado/pyrado/sampling/step_sequence.py

```diff
@@ -855,7 +855,9 @@ def recursive_wrapper(inp, **kwargs):
             data_dict[name] = data
 
         # Create new object
-        return StepSequence(**data_dict, rollout_info=rollout.rollout_info, continuous=rollout.continuous)
+        return StepSequence(
+            **data_dict, rollout_info=rollout.rollout_info, continuous=rollout.continuous
+        )  # pylint: disable=missing-kwoa
```

Comment:
maybe fixing the order would be better than disabling the warning? since there are only kwargs here, you could move **data_dict to the end

Reply:
I think the ”error“ was that the dictionary is generated dynamically, but I'll look at it again.

Reply:
I was right, but the disable thingy was on the wrong line.